lact-gui/Cargo.toml

```diff
@@ -15,7 +16,7 @@ gtk = { version = "0.7", package = "gtk4", features = ["v4_6", "blueprint"] }
 tracing = "0.1"
 tracing-subscriber = { version = "0.3", features = ["env-filter"] }
 anyhow = "1.0"
-libadwaita = { version = "0.5.3", features = ["v1_4"] }
+libadwaita = { version = "0.5.3", features = ["v1_4"], optional = true }
```

Comment:
I think it would be convenient to alias `libadwaita` to `adw`, similar to how `gtk4` is renamed to just `gtk`. Makes it easier to type out full symbol paths

Reply:
should be fixed as well